client/src/pages/UserProfile.js

```diff
@@ -0,0 +1,234 @@
+// Forewarning, I have no idea what I'm doing when it comes to React
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
```

Comment:
This can be in the import statement below

Reply:
I forgot to check the imports, an addon for VSCode I have automatically adds the imports when I use something new, I'll change that.